src/App.vue

```diff
@@ -14,7 +14,7 @@
 import Sidebar from '@/components/Sidebar.vue';
 import Vue from 'vue';
 
-import apps from './appregistry.json';
+const apps = require('./appregistry.json');
```

Comment:
Is this being done to satisfy the linter complaining about not specifying `resolveJsonModule`? If so, the following addition to `tsconfig.json` will fix this:

```
  "compilerOptions": {
    ...
    "resolveJsonModule": true,
```

Which I've tested locally and does work.

Reply:
I don't quite remember, but I think it had to do with the fact that we are really loading `appregistry.json` at runtime, hence the need for `require()`. Or maybe I just misunderstood what the errors were. In any case, we will be retooling this mechanism very soon in order to get Netlify to build properly linked DNSed vis apps, so I'm ok with leaving this as is and not inflating the ruleset on its account.